### DISCUSSION TITLE
Adding `eglot` as backend for IDE features?

### DISCUSSION THREAD

**Discussion by cole-jacobs** [syl20bnr/spacemacs#15914] [Ideas]
With eglot being included with new versions of emacs, I'm wondering if there is any desire for eglot adoption in spacemacs layers?

**Comment by lebensterben** [comment]
I fully support this idea but I don't have any extra capacity to do this in near terms.

**Comment by smile13241324** [comment]
There is an eglot layer available please feel free to use it and build upon it.
